src/get_games.py

```diff
@@ -40,38 +40,38 @@ def save_to_file(game,
         save_dir (str): Folder in which the games will be saved.
         save_file (str): File in which the games will be saved.
     Returns:
+    -------
         None
 
     """
     filename = os.path.join(save_dir, save_file + '.pgn')
-    with open(filename, 'w') as file:
+    with open(filename, 'a') as file:
         file.write("".join(game))
 
 
 def download_games(name: str,
                    db_dir: str = 'resources',
-                   pref_type: str = None,
-                   initial_time: int = None,
-                   latest_time: int = None,
+                   perf_type: str = None,
```

Comment:
This variable name does not match the documentation below.

Reply:
Documentation was wrong.